### PR TITLE
Fixes : Mapthon Query , Class Docs , Config 

### LINKED ISSUE #63: Unit test cases  for user group report Queries
Unit test required for /osm-users/statistics/ endpoint, to make sure that class is generating right query

### LINKED ISSUE #65: Update API docs
Currently the API docs reference "osm stats". All occurrences should use "galaxy" instead.

### LINKED ISSUE #67: Remove config object from multiple paths and move config to src
```API/osm_users.py
API/__init__.py
src/galaxy/__init__.py
src/galaxy/app.py
```
these are the files that created a config object on each, The idea is to initialize config in app.py only and use it from outside if needed for that we need to move the config file to src to avoid folder pointing issue  
cc : @JorgeMartinezG

### LINKED ISSUE #68: Mapathon end point producing strange results
To fix this bug we need to Update Query Builder's hashtag query producing function ! @omranlm  identified the issue in the query . We need to update it and check the associated affected endpoint and make the test work.
For that result and query generated from the builder needs to be verified.
cc : @JorgeMartinezG

### PR DESCRIPTION
**Pull request by itskshitiz321**
#### Completed : 

- Updated  Class Docs to use Galaxy 
- Added unit test for user group report query and modified uni tests after mapathon hashtag query fixing
- added config.py and initialized config variable under that file and made other files use same config
- Done with Hashtag Query builder fixing

### Worked on Issue

- Resolve #68 
- Resolve #67
- Resolve #65 
- Resolve #63

### CODE REVIEW

**Comment by itskshitiz321** [comment]
Now config file is inside src and initialized in config.py only cc: @JorgeMartinezG 
image

**Review by omranlm — CHANGES_REQUESTED** [review]
In this section, we should have semicolon, not a space as it is `("tags" -> \'hashtags\') ~~`
image

File: tests/src/test_app.py [hunk 1]
```diff
@@ -89,30 +89,30 @@ def teardown_module(module):
...
...6 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-1381 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11203 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10681 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8055 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8732 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11193 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-7305 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11210 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10985 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10988 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11190 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6658 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-5644 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10913 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6495 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-4229 %\' OR ("tags" -> \'comment\') ~~ \'%mapandchathour2021;%\')\n    )\n    SELECT (each(tags)).key AS feature, action, count(distinct id) AS count FROM osm_element_history AS t2, t1\n    WHERE t1.changeset_id = t2.changeset\n    GROUP BY feature, action ORDER BY count DESC\n    '
```
Comment (omranlm) [inline tests/src/test_app.py:104]: here, we should have semicolon, not a space as it is ("tags" -> \'hashtags\') ~~

File: tests/src/test_app.py [hunk 2]
```diff
@@ -89,30 +89,30 @@ def teardown_module(module):
...
...ags\') ~~ \'%mapandchathour2021 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11224 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10042 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-9906 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-1381 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11203 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10681 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8055 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8732 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11193 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-7305 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11210 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10985 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10988 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11190 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6658 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-5644 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10913 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6495 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-4229 %\' OR ("tags" -> \'comment\') ~~ \'%mapandchathour2021;%\')\n            '
```
Comment (omranlm) [inline tests/src/test_app.py:115]: here, we should have semicolon, not a space as it is ("tags" -> \'hashtags\') ~~

File: tests/src/test_app.py [hunk 3]
```diff
@@ -121,18 +121,18 @@ def test_mapathon_total_contributor_mapathon_query_builder():
...
..._id,\n            username,\n            count(distinct id) AS count\n        FROM osm_element_history AS t2, t1\n        WHERE t1.changeset_id    = t2.changeset\n        GROUP BY feature, user_id, username\n    ),\n    T3 AS (\n        SELECT user_id,\n            username,\n            SUM(count) AS total_buildings\n        FROM T2\n        WHERE feature = \'building\'\n        GROUP BY user_id, username\n    )\n    SELECT user_id,\n        username,\n        total_buildings,\n        public.tasks_per_user(user_id,\n            \'11224,10042,9906,1381,11203,10681,8055,8732,11193,7305,11210,10985,10988,11190,6658,5644,10913,6495,4229\',\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\',\n            \'MAPPED\') AS mapped_tasks,\n        public.tasks_per_user(user_id,\n            \'11224,10042,9906,1381,11203,10681,8055,8732,11193,7305,11210,10985,10988,11190,6658,5644,10913,6495,4229\',\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\',\n            \'VALIDATED\') AS validated_tasks,\n        public.editors_per_user(user_id,\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\') AS editors\n    FROM T3;\n    '
```
Comment (omranlm) [inline tests/src/test_app.py:129]: here, we should have semicolon, not a space as it is ("tags" -> \'hashtags\') ~~

**Pull request merged**